docs/edit/manifest.md

```diff
@@ -29,28 +57,9 @@ tests/:
       Test_FeatureE: *5_6_and_someid_backports
 ```
 
-### Implementation
-
-- Manifests files are inside `manifests/` folder, one per team (each library, agent, ASM rules file, any php extension...): `manifests/golang.yml`, `manifest/agent.yml`...
-- Each team has ownership of its manifest file
-- Manifest file are validated using JSON schema in system tests CI
-- An error pops if a manifest file refers to a file/class that does not exists
-
-## Supported features
-
-- declaring a skip reason (bug, flaky, irrelevant, missing_feature) for entire file
-- declaring released version (or a skip reason) for a class
-- declaring released version (or a skip reason) for a class, with details for weblog variants
+#### Notes
 - The wildcard `*` is supported for weblog declarations
```

Comment:
```suggestion
- The wildcard `*` is supported for weblog declarations. This will associate missing_feature/bug/flak/etc. marking to all unspecified weblog variables.
```